runbotics/runbotics-desktop/src/action/folder/folder.test.ts

```diff
@@ -202,14 +210,61 @@ describe('FolderActionHandler', () => {
             fs.mkdirSync(`${tempFolderPath}${path.sep}${testFolderName}`, { recursive: true });
             // create subfolder with one element
             fs.mkdirSync(`${tempFolderPath}${path.sep}${testFolderName}${path.sep}${folderByPath}`);
-            
+
             const folderContent = await folderActionHandler.displayFiles({
                 path: `${tempFolderPath}${path.sep}${testFolderName}`,
             });
-            
+
             expect(Array.isArray(folderContent)).toBeTruthy();
             expect(folderContent).toHaveLength(1);
             expect(folderContent).toContain(folderByPath);
         });
     });
+
+    describe('Create folder', () => {
+        it('Should throw error when name is not provided', async () => {
+            const params: FolderCreateActionInput = {
+                name: undefined,
+                path: cwd
+            };
+
+            await expect(folderActionHandler.createFolder(params)).rejects.toThrowError(
+                'Cannot create directory if name is not provided'
+            );
+        });
+
+        it('Should throw error if accessing non-existent folder', async () => {
+            const path = 'iDontExist';
+            const params: FolderCreateActionInput = {
+                name: 'test',
+                path: path
+            };
+
+            await expect(folderActionHandler.createFolder(params)).rejects.toThrowError(`Directory not found: ${path}`);
+        });
+
+        it('Should throw error if path is not provided', async () => {
```

Comment:
based on my previous comment, this test case is not necessary when we don't expect user to provide path